pkg/pgmodel/common/schema/schema.go

```diff
@@ -5,22 +5,10 @@
 package schema
 
 const (
-	Tag              = "ps_tag"
-	Prom             = "prom_api"
-	Data             = "prom_data"
-	Info             = "prom_info"
-	Exemplar         = "prom_data_exemplar"
-	Ext              = "_prom_ext"
-	Catalog          = "_prom_catalog"
-	PromscaleCatalog = "_ps_catalog"
-	Timescale        = "public"
+	Data     = "prom_data"
+	Exemplar = "prom_data_exemplar"
```

Comment:
We can get rid of these now, right?

Reply:
I would like to, but I'm honestly not sure exactly how. The pattern that was used with these was slightly different to the other usages which I've already replaced. In particular, these usages all/mostly use `pgx.Identifier`, in combination with some other item, like a table name or similar. I'm not entirely sure a) if we want to replace all of those, or b) how it should best be done. Please give me your thoughts.

Reply:
ok, I'll look...

Reply:
I think we can just use the strings in pgx.Identifier{} directly. If we want to keep constants, we should rename them to make sure they are only used as appropriate

Reply:
Ah, the `pgx.Identifier` pattern is just there to make sure that the table name is properly escaped as a PostgreSQL identifier. I'll fix...